pxr/usdImaging/lib/usdImaging/delegate.cpp

```diff
@@ -188,7 +189,7 @@ UsdImagingDelegate::_IsDrawModeApplied(UsdPrim const& prim)
     // Draw mode is only applied on models that are components, or which have
     // applyDrawMode = true.
     UsdModelAPI model(prim);
-    bool applyDrawMode = false;
+    bool applyDrawMode = !prim.IsLoaded();
```

Comment:
We only ever call _IsDrawModeApplied from _AdapterLookup, and in the case of this feature, we early-out before we get to that call.  I'd revert this line; it shouldn't affect your feature.